### PR TITLE
feat: add sell data Tx and query a datasale CLI

### PR DESCRIPTION
**Pull request by inchori**
- Add sell data transaction CLI.
- Add query a data sale CLI.
- When test it manually, it works well.

### CODE REVIEW

File: x/datadeal/client/cli/queryDataSale.go [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+			}
+
+			queryClient := types.NewQueryClient(clientCtx)
+
+			dealID, err := strconv.ParseUint(args[0], 10, 64)
+			if err != nil {
+				return err
+			}
+
+			verifiableCID, err := base64.StdEncoding.DecodeString(args[1])
```
Comment (gyuguen) [inline x/datadeal/client/cli/queryDataSale.go:31]: As discussed in other PRs, you can use string as it is.

File: x/datadeal/client/cli/txSellData.go [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+			}
+
+			sellerAddress := clientCtx.GetFromAddress().String()
+
+			dealID, err := strconv.ParseUint(args[0], 10, 64)
+			if err != nil {
+				return err
+			}
+
+			verifiableCID, err := base64.StdEncoding.DecodeString(args[1])
```
Comment (gyuguen) [inline x/datadeal/client/cli/txSellData.go:32]: This is same too,

**Review by gyuguen — COMMENTED** [review]
Thanks.
I left a few comment

File: x/datadeal/client/cli/txSellData.go (hunk 2 above)
Comment (inchori) [reply x/datadeal/client/cli/txSellData.go]: Fixed in [9ed50b9](https://github.com/medibloc/panacea-core/pull/422/commits/9ed50b907de38b143625cb4846520db63da6d1b0).

**Review by gyuguen — APPROVED** [review]
LGTM 👍

File: x/datadeal/client/cli/txSellData.go [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+
+			sellerAddress := clientCtx.GetFromAddress().String()
+
+			dealID, err := strconv.ParseUint(args[0], 10, 64)
+			if err != nil {
+				return err
+			}
+
+			msg := types.NewMsgSellData(dealID, args[1], sellerAddress)
+			if err = msg.ValidateBasic(); err != nil {
```
[the remark this answers is not in the document]
Comment (inchori) [reply x/datadeal/client/cli/txSellData.go:32]: Fixed in [e242ea8](https://github.com/medibloc/panacea-core/pull/422/commits/e242ea8adc675e716397e4a7795d49e3bc2ef6c4).

**Review by H4NLee — APPROVED** [review]
LGTM

**Pull request merged**